src/utils/checkDuplication.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} str
+ */
+
+const checkDuplication = function checkDuplicationFromString(str) {
+  const charSet = new Set(str);
+
+  if (str.length !== charSet.size) {
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return str.length !== charSet.size;
```
return문을 단순화하면 함수가 좀 더 간결해질수 있을 것 같아요!